neo/tools/compilers/fontcodec/BMfont.cpp

```diff
@@ -0,0 +1,128 @@
+/*
+ * BMfont.cpp
+ *
+ *  Created on: 3 de febr. 2018
+ *      Author: Biel Bestué de Luna
+ *
+ *      this file is GPLv3
+ */
+
+//#include <rapidxml>
+#include "../../libs/rapidxml/rapidxml-1.13/rapidxml.hpp"
```

Comment:
additionally to this you need to include rapidxml_utils.hpp like this: 

#include "../../libs/rapidxml/rapidxml-1.13/rapidxml_utils.hpp"